nextjs-13/src/repositories/newsletter.repository.ts

```diff
@@ -12,17 +12,29 @@ const deleteNewsletter = async (id: string, userId: string) =>
 const updateNewsletter = async (args: Prisma.NewsletterUpdateManyArgs) =>
   dbClient.newsletter.updateMany(args);
 
-const queryNewsletterByUserId = async (id: string) =>
-  dbClient.newsletter.findMany({
-    where: { userId: id },
+const queryNewsletters = async (userId: string, ids = []) => {
```

Comment:
Avoid plurals https://nimblehq.co/compass/development/code-conventions/#convention-15 👮‍♀️ 

Prefer: 

```suggestion
const queryNewsletterList = async (userId: string, ids = []) => {
```

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/121/commits/607cd3a298f43bac5c4e592004a6a7cd15f0581e#diff-7f62d1d6ff486da9e5be846b0de3dc572e91704e2e913d9aa47ab9e57090f37bR15)